tests/sandbox-anchor.js

```diff
@@ -1,14 +1,255 @@
 const anchor = require('@project-serum/anchor');
+const secp256k1 = require('secp256k1');
+const {keccak_256 } = require('js-sha3');
+const { snakeCase } = require('snake-case');
+const { sha256 } = require("js-sha256");
+
+const {struct, u8, u16, u32, blob} = require("@solana/buffer-layout");
+const {Buffer} = require('buffer');
+
+function MsgsToInstructionData(messages, instructionIndex) {
```

Comment:
I have not seen PascalCase in function names. What convention do you use for names?

Reply:
Well, it's hard for me to evaluate, but is it possible to specify the return type?

Reply:
Well, the name of the function must contain a verb